src/fmu/dataio/dataio.py

```diff
@@ -469,20 +484,97 @@ def __post_init__(self) -> None:
         self._validate_content_key()
         self._validate_and_establish_fmucontext()
         self._validate_workflow_key()
-        self._update_globalconfig_from_settings()
 
         # check state of global config
         self._config_is_valid = global_configuration.is_valid(self.config)
         if self._config_is_valid:
             # TODO: This needs refinement: _config_is_valid should be removed
             self.config = global_configuration.roundtrip(self.config)
 
+        self._classification = self._get_classification()
+        self._rep_include = self._get_rep_include()
+
         self._establish_pwd_rootpath()
         logger.info("Ran __post_init__")
 
+    def _get_classification(self) -> enums.AccessLevel:
```

Comment:
Should probably (in future) rename this to `enum.Classification`. Particularly when the `ssdl.access_level` eventually goes away.

Reply:
Agree 👍 will make an issue